### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by jason--liu** [org-roam/org-roam-ui#261]
**Describe the bug**
On the latest org-roam-ui branch, in org-roam-ui, underline becomes subscript. Even if I add the following line in the file
```
#+OPTIONS: ^:nil 
```
But, if I export to a html file, the above setting is working and underline is underline not subscript.

**To Reproduce**
It's 100% reproducible. It's the same when you open any file.

**Expected behavior**
underline is underline, do not become subscript.

**Screenshots**
![image](https://user-images.githubusercontent.com/8001473/187195259-e53424cd-50e8-42e3-856d-0b13d78362df.png)

**What browser were you using?**
Chrome

Thanks for your help

**Comment by tefkah** [comment]
@rasendubi  do you know if this is a uniorg issue or an org-roam-ui issue?

**Comment by rasendubi** [comment]
It's a uniorg one. There are two ways to enable this behavior in org-mode:
via a variable or via options. Neither way is currently supported by uniorg

**Comment by jason--liu** [comment]
Is there any workaround method that can be fixed? Thank you. @rasendubi

**Comment by rasendubi** [comment]
Not really. The easiest way to fix this is to add support for `org-use-sub-superscripts` in uniorg. If someone is interested in implementing this, I can provide some guidance

**Comment by seokbeomKim** [comment]
@rasendubi, This ticket is quite old, but could you share the guide?

**Comment by rasendubi** [comment]
oh, org-use-sub-superscripts is already implemented now, so the only thing
left is to pass correct configuration from org-roam-ui

See
https://github.com/rasendubi/uniorg/blob/11ef10311b05ebd4d0dea80a7f4c51d4aa82edcf/packages/uniorg-parse/src/parse-options.ts#L14
for config documentation

**Comment by seokbeomKim** [comment]
@rasendubi Thank you for sharing!
@tefkah I cannot find any documentation to configure uniorg's ParseOption. Aren't there any configuration interfaces for this yet in org-roam-ui?

**Comment by sgript** [comment]
Any updates on this?
